vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -42,14 +44,16 @@ type EsxVmdkCmd struct {
 }
 
 const (
-	commBackendName string = "vsocket"
-	maxRetryCount          = 5
+	commBackendName        string = "vsocket"
+	maxRetryCount                 = 5
+	protocolCurrentVersion        = "2"
 )
 
 // A request to be passed to ESX service
 type requestToVmci struct {
 	Ops     string     `json:"cmd"`
 	Details VolumeInfo `json:"details"`
+	Version string     `json:"version"`
```

Comment:
so will it break compat with existng version ? Do you think an optional "version" is better, and the lack of version indicates V1 ?